rocky/onboarding/view_helpers.py

```diff
@@ -3,6 +3,8 @@
 from django.utils.translation import gettext_lazy as _
 from tools.view_helpers import StepsMixin
 
+ONBOARIDNG_PERMS = ["tools.can_scan_organization", "tools.can_set_clearance_level", "tools.can_enable_disable_boefje"]
```

Comment:
```suggestion
ONBOARIDNG_PERMISSIONS = ["tools.can_scan_organization", "tools.can_set_clearance_level", "tools.can_enable_disable_boefje"]
```

Reply:
You're also better of with a tuple instead of a list

Reply:
I see there is a typo also in ONBOARDING...fix it in [c82ccb9](https://github.com/minvws/nl-kat-coordination/pull/1275/commits/c82ccb92ae3e36e8f2dcd53edcc24a89d98e7804)